tests/test_sklearn_models.py

```diff
@@ -20,79 +20,69 @@ def iter_models(*modules):
     "net",
     list(iter_models("svm", "linear_model", "naive_bayes", "tree")),
 )
-def test_save_load(tmpdir, net):
-    tiledb_array = os.path.join(tmpdir, "test_array")
-    model = net()
-    tiledb_sklearn_obj = SklearnTileDB(uri=tiledb_array)
-    tiledb_sklearn_obj.save(model=model)
-    loaded_model = tiledb_sklearn_obj.load()
-    assert all([a == b for a, b in zip(model.get_params(), loaded_model.get_params())])
-
-
-@pytest.mark.parametrize(
-    "net",
-    list(iter_models("svm", "linear_model", "naive_bayes", "tree")),
-)
-def test_preview(tmpdir, net):
-    # With model as argument
-    tiledb_array = os.path.join(tmpdir, "test_array")
-    model = net()
-    tiledb_sklearn_obj = SklearnTileDB(uri=tiledb_array, model=model)
-    assert type(tiledb_sklearn_obj.preview()) == str
-
-    # Without model as argumet
-    tiledb_sklearn_obj = SklearnTileDB(uri=tiledb_array)
-    assert type(tiledb_sklearn_obj.preview()) == str
-
-
-@pytest.mark.parametrize(
-    "net",
-    list(iter_models("svm", "linear_model", "naive_bayes", "tree")),
-)
-def test_file_properties(tmpdir, net):
-    model = net()
-    tiledb_array = os.path.join(tmpdir, "model_array")
-    tiledb_obj = SklearnTileDB(uri=tiledb_array)
-    tiledb_obj.save(model=model)
-
-    assert tiledb_obj._file_properties["ML_FRAMEWORK"] == "SKLEARN"
-    assert tiledb_obj._file_properties["STAGE"] == "STAGING"
-    assert tiledb_obj._file_properties["PYTHON_VERSION"] == platform.python_version()
-    assert tiledb_obj._file_properties["ML_FRAMEWORK_VERSION"] == sklearn.__version__
-    assert tiledb_obj._file_properties["PREVIEW"] == ""
-
-
-@pytest.mark.parametrize(
-    "net",
-    list(iter_models("svm", "linear_model", "naive_bayes", "tree")),
-)
-def test_file_properties_in_tiledb_cloud_case(tmpdir, net, mocker):
-    model = net()
-    tiledb_array = os.path.join(tmpdir, "model_array")
-
-    mocker.patch("tiledb.ml._cloud_utils.get_s3_prefix", return_value="")
-    mocker.patch(
-        "tiledb.ml.models.base.TileDBModel.set_cloud_uri", return_value=tiledb_array
-    )
-    mocker.patch("tiledb.ml._cloud_utils.update_file_properties")
-
-    tiledb_obj = SklearnTileDB(uri=tiledb_array, namespace="test_namespace")
-    tiledb_obj.save(model=model)
+class TestSklearnModel:
+    def test_save_load(self, tmpdir, net):
+        tiledb_array = os.path.join(tmpdir, "test_array")
+        model = net()
+        tiledb_sklearn_obj = SklearnTileDB(uri=tiledb_array)
+        tiledb_sklearn_obj.save(model=model)
+        loaded_model = tiledb_sklearn_obj.load()
+        assert all(
+            [a == b for a, b in zip(model.get_params(), loaded_model.get_params())]
```

Comment:
`zip()` takes the shortest of the inputs so this will succeed if `loaded_model.get_params()` is for example empty . 
Either assert for equal length before the zip or use [zip_longest](https://docs.python.org/3/library/itertools.html#itertools.zip_longest)